app/assets/javascripts/application.js

```diff
@@ -12,16 +12,21 @@
 //= require angular-ui-bootstrap
 //= require angular-ui-bootstrap-tpls
 //= require angular-sanitize
+//= require angular-ui-router
+//= require angular-rails-templates
 //= require moment
 //= require moment-strftime/build/moment-strftime.min
 //= require moment-timezone
 //= require sprintf
 //= require numeral
 //= require miq_api
 //= require miq_angular_application
+//= require miq_angular_configuration
+//= require_tree ./components/
+//= require_tree ./controllers/
 //= require_tree ./directives/
+//= require_tree ./factories/
```

Comment:
@himdel Are there any articles on why `services` would be better than `factories`?

The advantage of using factories is that they are preloaded with data from the REST API which can be consumed by the controller right away.

Also, SSUI is using factories... I believe probably for the same reason.